### ISSUE TITLE
GENOMIC in Library Sources Advanced Search field returns false positives

### ISSUE THREAD

**Issue by ropolomx** [BeeBiome-consortium/beebiome-data-portal#9]
Searching the term GENOMIC in the Library Sources Advanced Search field returns results that contain the word "GENOMIC", but that do not refer to single genomes. Examples include "METAGENOMIC", or "METAGENOMIC, TRANSCRIPTOMIC". https://beebiome.info/search

**Comment by ropolomx** [comment]
FYI @vrechdelaval

**Comment by ropolomx** [comment]
FYI @vrechdelaval

**Comment by vrechdelaval** [comment]
@ropolomx Indeed, the current search is a simple text search. The API builds the requests and queries the db, so you have to modify the API if you want to do that. To be discussed with users to know what they think about it.

**Comment by ropolomx** [comment]
Thanks for your insights @vrechdelaval . @greysaruman is working on this. As a user of the portal myself, it would certainly be nice to see that when I enter GENOMIC in Advanced Search I only get metadata from single genome entries.

**Comment by marcrr** [comment]
A drop down menu makes sense, but that means first implementing a controlled vocabulary on this metadata.
